AU2/html_components/SimpleComponents/Checkbox.py

```diff
@@ -6,11 +6,12 @@
 class Checkbox(HTMLComponent):
     name: str = "Checkbox"
 
-    def __init__(self, identifier: str, title: str, checked: bool=False):
+    def __init__(self, identifier: str, title: str, checked: bool=False, force_default: bool=False):

```

Comment:
force this to be a keyword arg for clarity (there's an argument `checked` should have been as well, but that's beyond this PR's scope)

```suggestion
    def __init__(self, identifier: str, title: str, checked: bool=False, *, force_default: bool=False):

```